viz.yaml

```diff
@@ -117,6 +121,16 @@ fetch:
       view-limits: "view-limits-mobile"
     mimetype: application/zip
   -
+    id: precip_shapefiles
+    location: cache/precip_shp/
+    reader: rds
```

Comment:
i haven't tried this stuff in a while, but will readData work on a directory when the reader is rds?

Reply:
I overlooked the reader big here. Was going to open and process all these files, but that seemed like too much to do in a fetcher and keeping the native files will let us update more carefully.

Reply:
looks like it'll be the `folder` reader, when we get to this.

Reply:
Will I need a different mimetype for the output of the fetch too?

Reply:
reader overrides mimetype, so no mimetype is needed